### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Updates Rollup as Chromium is now at Node 20.

### CODE REVIEW

File: tests/network/test_network.py [hunk 1]
```diff
@@ -20,6 +20,12 @@
                           subscribe, wait_for_event)
 
 
+def compute_response_headers_size(headers) -> int:
```
Comment (Lightning00Blade) [inline tests/network/test_network.py:23]: Drive-by: extract to top.

**Review by OrKoN — APPROVED** [review]
LGTM % precommit
